gulpfile.js

```diff
@@ -84,20 +86,19 @@ function watch(){
 
 function getSharedPortalComponentsLink(){
     let sharedComponentPath = path.resolve(__dirname, 'node_modules/@hms-dbmi-bgm/shared-portal-components');
-    let isLinked = false;
+    const origPath = sharedComponentPath;
 
-    try { // Get exact path to dir, else leave. Used to avoid needing to webpack dependency itself.
-        for (var i = 0; i < 10; i++) { // Incase multiple links.
-            sharedComponentPath = fs.readlinkSync(sharedComponentPath);
-            isLinked = true;
-        }
-    } catch (e){
-        // ... not linked
-    }
```

Comment:
This existing code didn't work with `nvm` that well, it would grab the correct path of the symlink once, but then if that correct path was also a symlink, it wouldn't necessarily find the real path. Using `fs.realpathSync(sharedComponentPath);` appears to be more stable. If works well will migrate to CGAP.